src/test/groovy/edu/ie3/datamodel/io/factory/input/LineInputFactoryTest.groovy

```diff
@@ -174,4 +174,42 @@ class LineInputFactoryTest extends Specification implements FactoryTestHelper {
 		"{ \"type\": \"LineString\", \"coordinates\": [[7.411111, 51.49228],[7.411111, 51.49228],[7.411111, 51.49228],[7.411111, 51.49228]]}" | _
 		"{ \"type\": \"LineString\", \"coordinates\": [[7.411111, 51.49228],[7.411111, 51.49228],[7.311111, 51.49228],[7.511111, 51.49228]]}" | _
 	}
+
+	def "A LineInputFactory should parse a valid LineInput with different double double quoted geoPosition strings correctly"() {
```

Comment:
When the above comments are addressed, please make this a test, that works on the String manipulation method and not on the factory, as the factory will only be calling the aforementioned method.